build_tools/create_log_index.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+from pathlib import Path
+import sys
+import platform
+import argparse
+
+
+def log(*args):
+    print(*args)
+    sys.stdout.flush()
+
+
+def is_windows():
+    return platform.system().lower() == "windows"
+
+
+def normalize_path(p: Path) -> str:
+    return str(p).replace("\\", "/") if is_windows() else str(p)
+
+
+def index_log_files(build_dir: Path, amdgpu_family: str):
+    log_dir = build_dir / "logs"
+    index_file = log_dir / "index.html"
+    indexer_path = build_dir / "indexer.py"
```

Comment:
This still relies on the `indexer.py` file being downloaded from https://raw.githubusercontent.com/joshbrunty/Indexer/6d8cbfd15d3853b482e6a49f2d875ded9188b721/indexer.py ahead of time, right? We should still fork that file and customize it instead of having CI jobs download from github.

Fine for now, to keep this PR small, but could add a TODO.

Reply:
I was planning to do that actually - will add TODO for that.